### PR TITLE
Fix c02ef3e456: Special case of Var 0x88 was not handled for NRT Acti…

### PR DESCRIPTION
**Pull request by Eddi-z**
…on 7 conditions

### CODE REVIEW

**Comment by Eddi-z** [comment]
it's a bit unclear why there are 4 condition types for NRT checks, seems 2 would be enough

**Review by LordAro — DISMISSED** [review]

File: src/newgrf.cpp [hunk 1]
```diff
@@ -6630,10 +6632,6 @@ static void SkipIf(ByteReader *buf)
 				break;
 			case 0x10: result = GetRoadTypeByLabel(BSWAP32(cond_val)) != INVALID_ROADTYPE;
 				break;
-			case 0x11: result = GetRoadTypeByLabel(BSWAP32(cond_val)) == INVALID_ROADTYPE;
-				break;
```
Comment (frosch123) [inline src/newgrf.cpp:6634]: I think this stuff is correct and should be kept.
There are also separate translation tables for road and tram.

But who knows.

**Review by frosch123 — CHANGES_REQUESTED** [review]

File: src/newgrf.cpp (hunk 1 above)
Comment (Eddi-z) [reply src/newgrf.cpp:6634]: If they should be kept as a separation between Road and Tram, they should probably be extended with actually different checks.

**Comment by frosch123** [comment]
Part of this was already fixed in 7f351fd7c1e853918d2ce7bfd1c9c6c5a2af84ef.
The rest is incorporated into #8382.

**Pull request closed without merging**